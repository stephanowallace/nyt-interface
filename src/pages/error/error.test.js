import React from 'react';
import { shallow } from 'enzyme';
import Error from '.';

describe('Error', () => {
  it('renders default state', () => {
    const component = shallow(<Error />);
    expect(component).toMatchSnapshot();
  });
});
