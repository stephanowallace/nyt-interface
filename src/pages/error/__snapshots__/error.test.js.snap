// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renders default state 1`] = `
<CenterContent>
  <PageHeader
    title="Error"
  />
  <div
    className="content"
  >
    <test-file-stub />
    <Paragraph
      className=""
    >
      Something unexpected happened
    </Paragraph>
    <SecondaryText
      className=""
    >
      Please, try again later
    </SecondaryText>
  </div>
</CenterContent>
`;
